android/app/src/main/res/layout/tracksactvity_item.xml

```diff
@@ -75,6 +75,7 @@
                     style="@style/Base.TextAppearance.AppCompat.Subhead"
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
+                    android:visibility="gone"
```

Comment:
So when are we showing this?

Reply:
@iamareebjamal It was already managed at the other end. The visibility check is already present in the SessionViewHolder file